frontend/src/components/pages/WaitlistExperience/WaitlistFooter.tsx

```diff
@@ -63,6 +63,7 @@ const WaitlistFooter = ({
       <Button
         ref={nextBtnRef}
         width={{ sm: "95vw", md: "45vw", lg: "auto" }}
+        mr={{ sm: 0, lg: 4 }}
```

Comment:
Also not related, just adds some margin (exists on the other, RegistrationFooter component).